src/components/Profile/components/SchedulePanel/index.jsx

```diff
@@ -240,29 +122,12 @@ class GordonSchedulePanel extends Component {
   }
 
   render() {
-    const replaced = this.state.description.replace(urlRegex({ strict: false }), function (url) {
-      if (url.split('://')[0] !== 'http' && url.split('://')[0] !== 'https') {
-        return '<a target="_blank" rel="noopener" href="https://' + url + '">' + url + '</a>';
-      } else {
-        return '<a target="_blank" rel="noopener" href="' + url + '">' + url + '</a>';
-      }
-    });
+    const replaced = this.state.description;
```

Comment:
```suggestion
    const replaced = this.state.description;
```
`replaced` is only used in 1 location. Is there a reason it needs to be a variable?